tests/baselines/reference/outfile-concat.js

```diff
@@ -0,0 +1,26 @@
+var s = "Hello, world";
+console.log(s);
+console.log(f());
+function f() {
+    return "JS does hoists";
+}
+//# sourceMappingURL=first-output.js.map
+var N;
+(function (N) {
+    function f() {
+        console.log('testing');
+    }
+    f();
+})(N || (N = {}));
+var C = (function () {
+    function C() {
+    }
+    C.prototype.doSomething = function () {
+        console.log("something got done");
+    };
+    return C;
+}());
+//# sourceMappingURL=second-output.js.map
+var c = new C();
+c.doSomething();
+//# sourceMappingURL=third-output.js.map
```

Comment:
We should probably avoid printing these interstitial `sourceMappingURL` comments in concatenated files. They have the potential to seriously confuse tools, should the files actually exist on disk.

Reply:
Minimally, we should be emitting a bundle-wide one on the last line, to override the one from the last concatenated file for well-behaved tools.